x-pack/test/functional/es_archives/endpoint/resolver/api_feature/data.json

```diff
@@ -0,0 +1,15462 @@
+{
```

Comment:
maybe gzip and/or shrink the size of this file? all the fields aren't required for the tests

Reply:
I'll gzip it.